multivac/gather_job_data.py

```diff
@@ -162,6 +164,53 @@ def gather_data(self):
                 )
             self.gathered_data[job_id] = gathered_job_data
 
+    def put_to_db(self):
+        bucket = os.getenv('INFLUX_BUCKET')
+        org = os.getenv('INFLUX_ORG')
+
+        write_api = influx_connector()
+
+        for job in list(self.gathered_data.keys()):
+            curr_job = self.gathered_data[job]
+
+            time_to_datetime = datetime.fromisoformat(
+                f"{curr_job['queued_at'].rstrip('Z')}+00:00")
+            time_to_unix = datetime.timestamp(time_to_datetime)
+            time = int(time_to_unix * 1000000000)
+
+            # status_value = 1 if curr_job['status'] == 'success' else 0
+
+            tags = {
+                'job_name': curr_job['job_name'],
+                'branch': curr_job['branch'],
+                'commit_sha': curr_job['commit_sha'],
+                'platform': curr_job['platform'],
+                'runner_label': curr_job['runner_label']
+            }
+            if 'runner_version' in curr_job.keys():
+                tags['runner_version'] = curr_job['runner_version']
+            if 'runner_name' in curr_job.keys():
+                tags['runner_name'] = curr_job['runner_name']
+            if 'failure_type' in curr_job.keys():
+                tags['failure_type'] = curr_job['failure_type']
+
+            fields = {
+                'status': curr_job['status'],
+                'started_at': curr_job['started_at'],
+                'completed_at': curr_job['completed_at'],
+            }
+
+            data = {
+                'measurement': f'Job {job}',
+                'tags': tags,
+                'fields': fields,
+                'time': time
+            }
+            write_api.write(bucket, org, [data])
+
+    def build_influx_query(self):
+        pass
```

Comment:
if it's a matter of next issue, maybe we shouldn't leave a stub here

Reply:
fixed